docs/csharp/language-reference/configure-language-version.md

```diff
@@ -22,6 +22,14 @@ If you must specify your C# version explicitly, you can do so in several ways:
 >
 > To know what language version you're currently using, put `#error version` (case sensitive) in your code. This makes the compiler report a compiler error, CS8304, with a message containing the compiler version being used and the current selected language version. See [#error (C# Reference)](preprocessor-directives.md#error-and-warning-information) for more information.
 
+## Why can't I select a different C# version?
+
+In Visual Studio, the option to change the language version through the UI may be disabled because the default version is aligned with the project's target framework (`TFM`). This default configuration ensures compatibility between language features and runtime support.
```

Comment:
grammar nit:

```suggestion
In Visual Studio, the option to change the language version through the UI might be disabled because the default version is aligned with the project's target framework (`TFM`). This default configuration ensures compatibility between language features and runtime support.
```